pwndbg/commands/context.py

```diff
@@ -8,6 +8,7 @@
 import ast
 import codecs
 import sys
+from io import open
```

Comment:
This makes it works with both Py2 and Py3 (see http://python-future.org/compatible_idioms.html#file-io-with-open ).